rust/query-stats/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+edition = "2021"
+name = "query_stats"
+version = "1.0.0"
+
+[lib]
+crate-type = ["cdylib"]
+
+[dependencies]
+candid = "0.10"
+ic-cdk = { git = "https://github.com/dfinity/cdk-rs", rev = "c01fc1741eb3fec2df0bb4e5c5ff54fa027a6091" }
```

Comment:
Do we have to pin a specific cdk commit? I'd prefer if we can use an official version